README.md

```diff
@@ -12,6 +12,7 @@ To see a miniature example of a MultiQC plugin, visit [example-plugin].
 
 Custom Plugins in this repository by Zavolan Lab, University of Basel, Switzerland:
 - [ALFA]
+- Tin-scores
```

Comment:
TIN stands for Transcript Integrity Number, all capitals.
Please link it also to: https://git.scicore.unibas.ch/zavolan_group/tools/tin_score_calculation